packages/common/mui/src/lib/line-chart/line-chart-tooltip/line-chart-tooltip.tsx

```diff
@@ -0,0 +1,110 @@
+import { TooltipProps } from 'recharts';
+import { Box, styled, useTheme } from '@mui/material';
+import { H5 } from '../../typography/typography';
+import { formatNumberToDigits } from '@notional-finance/helpers';
+import { LineChartConfigProps } from '../line-chart';
+
+// Added monthMapping to convert abbreviated month values to full month names
+const monthMapping: { [key: string]: string } = {
+  Jan: 'January',
+  Feb: 'February',
+  Mar: 'March',
+  Apr: 'April',
+  May: 'May',
+  Jun: 'June',
+  Jul: 'July',
+  Aug: 'August',
+  Sep: 'September',
+  Oct: 'October',
+  Nov: 'November',
+  Dec: 'December',
+};
+
+interface LineChartToolTipProps extends TooltipProps<number, string> {
+  lineConfig: LineChartConfigProps[];
+}
+
+export const LineChartToolTip = (props: LineChartToolTipProps) => {
+  const theme = useTheme();
+  const { payload, lineConfig } = props;
+
+  return (
+    <ToolTipBox>
+      <Item>
+        <H5 sx={{ fontSize: '12px', fontWeight: '600' }}>
```

Comment:
I'm not sure you need to override the fontSize and fontWeight here, I'd like to keep all the typography settings in the notional-theme file